pkg/process/config/config.go

```diff
@@ -578,6 +596,18 @@ func getHostname(ddAgentBin string) (string, error) {
 	return hostname, err
 }
 
+func getHostnameFromAgent(ddAgentClient pb.AgentClient) (string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
```

Comment:
The timeout value seems like a constant so I think it might need to be pulled up to the top of the file